TryFrom impls for augrim message to scabbard types

peterschwarz opened the pull request.

vaporos commented on services/scabbard/libscabbard/src/service/v3/consensus/process.rs:
```diff
@@ -0,0 +1,32 @@
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct ScabbardProcess(ServiceId);
+
+impl augrim::Process for ScabbardProcess {}
```
Add a use augrim::Process above

peterschwarz replied:
Fixed

peterschwarz commented on services/scabbard/libscabbard/src/service/v3/consensus/value.rs:
```diff
@@ -0,0 +1,38 @@
+use augrim::Value;
+
+#[derive(Clone)]
+pub struct ScabbardValue(Vec<u8>);
```
I believe this is a temporary value as we continue work with the other components, such as the publisher, etc

peterschwarz commented:
Moved the implementation of the try from to the appropriate store submodule. As this has been rebased on main which now includes the Consensus Action Runner, I've also added a feature to cover the consensus module more broadly, and includes the action runner-specific feature under it.
